src/a-runtime-console/kubernetes/ui/pipeline/input-action-dialog/input-action-dialog.component.html

```diff
@@ -4,18 +4,31 @@
   </modal-header>
 
   <modal-content>
-    <div class="input-action-message-panel">
-      <div *ngFor="let message of messageLines" class="input-action-message">
-        <p>{{message}}</p>
+    <ng-container *ngIf="this.jenkinsStatus; else jenkinsIdeled">
```

Comment:
I think there is a typo, not a problem though but good to be changed,
jenkinsIdeled to jenkinsIdled